src/document.js

```diff
@@ -0,0 +1,254 @@
+import React from 'react'
+
+export default class Document extends React.Component {
+
+  componentWillMount() {
+    let docUrl = this.props.docUrl
+    let split = docUrl.split('/')
+    let docId = split.pop()
+
+    this.baseUrl = split.join('/')
+
+    $.get(`${docUrl}/json`, (doc) => {
+      this.setState({doc: doc, loaded: true})
```

Comment:
I think `loaded` is a bit redundant. It's equivalent to `!! (this.state || {}).doc`.